.github/workflows/build_test.yml

```diff
@@ -0,0 +1,85 @@
+name: Build Sample React workflow
+on:
+  push:
+    branches:
+      - "*" 
```

Comment:
Is it worth to trigger workflow for push on all branches? Can we keep only major branches like master, staging, release candidates, release etc.